cmd/recommend.go

```diff
@@ -32,9 +32,10 @@ var recommendCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(recommendCmd)
 
-	recommendCmd.Flags().StringSliceVar(&recommendOptions.Images, "image", []string{}, "Container image list (comma separated)")
-	recommendCmd.Flags().StringSliceVar(&recommendOptions.Uselabels, "use-labels", []string{}, "User defined labels for policy (comma separated)")
-	recommendCmd.Flags().StringVarP(&recommendOptions.Outdir, "outdir", "o", "out", "output folder to write policies")
-	recommendCmd.Flags().StringVarP(&recommendOptions.Reportfile, "report", "r", "report.txt", "report file")
+	recommendCmd.Flags().StringSliceVarP(&recommendOptions.Images, "image", "i", []string{}, "Container image list (comma separated)")
+	recommendCmd.Flags().StringSliceVarP(&recommendOptions.UseLabels, "use-labels", "l", []string{}, "User defined labels for policy (comma separated)")
+	recommendCmd.Flags().StringVarP(&recommendOptions.UseNamespace, "use-namespace", "n", "", "User defined namespace value for policies")
```

Comment:
Sorry for this, but please remove use of the shorthands here.

These shorthands would be required for k8s based recommendations.

Reply:
done @nyrahul